core/src/main/resources/hudson/model/View/popup.jelly

```diff
@@ -0,0 +1,39 @@
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:s="/lib/form">
+  <div class="remodal new-job-popup" data-remodal-id="newjob" id="add-item-panel">
+    <form method="post" action="createItem" name="createItem" id="createItem">
+      <div class="header">
+        <div class="add-item-name">
+          <label for="name">${%ItemName.label}</label>
+          <input name="name" id="name" data-valid="false" type="text" tabindex="0" />
+          <div id="itemname-required" class="input-validation-message input-message-disabled">&#187; ${%ItemName.validation.required}</div>
+          <div id="itemname-invalid" class="input-validation-message input-message-disabled"></div>
+          <div id="itemtype-required" class="input-validation-message input-message-disabled">&#187; ${%ItemType.validation.required}</div>
+        </div>
+      </div>
+
+      <div id="items" class="categories flat" role="radiogroup" aria-labelledby="Items" data-valid="false" >
```

Comment:
👎   the implementation of `flat` mode only applies in the main screen (original new item option), not here. If you use this specific styles, if in the future these styles are changed, this will brake.